shiny/render/_data_frame_utils/_pandas.py

```diff
@@ -119,8 +122,10 @@ def serialize_pd_dtype(
         }
     elif t in {"datetime64", "datetime"}:
         t = "datetime"
-    elif t in {"timedelta", "timedelta64"}:
-        t = "timedelta"
+
+    # # Disable timedelta as narwhals does not support it
+    # elif t in {"timedelta", "timedelta64"}:
+    #     t = "timedelta"
```

Comment:
quick comment while scanning through this, but the equivalent is [Duration](https://narwhals-dev.github.io/narwhals/api-reference/dtypes/#narwhals.dtypes.Duration)

Reply:
Thank you!!